gio/src/list_store.rs

```diff
@@ -107,6 +107,16 @@ impl ListStore {
     }
 }
 
+impl<P: IsA<glib::Object>> std::iter::FromIterator<P> for ListStore {
```

Comment:
Could also implement it in addition for `&P` but looks good otherwise!

Reply:
Apparently not possible, causes conflicting impls

Reply:
That does not seem to work: 

```
110 | impl<P: IsA<glib::Object>> std::iter::FromIterator<P> for ListStore {
    | ------------------------------------------------------------------- first implementation here
...
120 | impl<P: IsA<glib::Object>> std::iter::FromIterator<&P> for ListStore {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `auto::list_store::ListStore`
    |
    = note: downstream crates may implement trait `glib::IsA<glib::Object>` for type `&_`
```